addon/templates/components/uni-header.hbs

```diff
@@ -1,7 +1,13 @@
 <div class="uni-header__wrapper">
-  {{#link-to logoRoute}}
-    {{inline-svg (if isTransparent "uniplaces_logo_white.svg" "uniplaces_logo.svg")}}
-  {{/link-to}}
+  {{#if logoUrl}}
+    <a href="{{logoUrl}}">
```

Comment:
I don't think it justifies creating an option to the target because this is our header - the logo sends us to the homepage - and I don't believe it makes sense to make the target blank ever.